recipes/libalsa/all/conanfile.py

```diff
@@ -89,4 +89,4 @@ def package_info(self):
         self.cpp_info.names["pkg_config"] = "alsa"
         self.cpp_info.names["cmake_find_package"] = "ALSA"
         self.cpp_info.names["cmake_find_package_multi"] = "ALSA"
-        self.env_info.ALSA_CONFIG_DIR = os.path.join(self.package_folder, "res", "alsa")
+        self.env_info.ALSA_CONFIG_DIR.append(os.path.join(self.package_folder, "res", "alsa"))
```

Comment:
`ALSA_CONFIG_DIR` cannot be a list of libraries: https://github.com/alsa-project/alsa-lib/blob/ddfc32abf5697de1618b9e7ffdf57a0f97013090/src/conf.c#L586-L596
It's used as a single top folder.

Reply:
@madebr Hmmm then we would have gstreamer and the plugins in the same recipe and they cannot be separate correct?

Reply:
So https://github.com/conan-io/conan-center-index/pull/7096 cannot work right?

Reply:
I think not with an unmodified libalsa.
I'm not a libalsa developer though. So perhaps you should ask upstream.

Reply:
@madebr Would something like this be acceptable?
https://github.com/conan-io/conan-center-index/pull/7104

Reply:
Looks reasonable, as long as the number of plugins don't explode.
Will you add it as libalsa? (Why the rename)
I would build the two subprojects in different folders.